analytical_engine/core/app/app_invoker.h

```diff
@@ -186,15 +187,18 @@ class AppInvoker {
   static void query_impl(std::shared_ptr<worker_t> worker,
                          const rpc::QueryArgs& query_args,
                          std::index_sequence<I...>) {
+    double t0 = grape::GetCurrentTime();
     worker->Query(
         ArgsUnpacker<typename std::remove_const<typename std::remove_reference<
             typename ArgTypeAt<I + 1, context_init_func_t>::type>::type>::
                          type>::unpack(query_args.args(I))...);
+    double t1 = grape::GetCurrentTime();
+    LOG(INFO) << "Query time: " << t1 - t0;
```

Comment:
Adding the unit for less confusing:

LOG(INFO) << "Query time: " << t1 - t0 << " seconds";